packages/cli/src/rpc/navie/suggest.ts

```diff
@@ -0,0 +1,31 @@
+import { NavieRpc } from '@appland/rpc';
+import { RpcHandler } from '../rpc';
+import { INavieProvider } from '../explain/navie/inavie';
+
+// We don't want to support context lookups
+const NOP = () => Promise.resolve([]);
+
+function getSuggestions(
+  navieProvider: INavieProvider,
+  threadId: string
+): Promise<NavieRpc.V1.Suggest.Response> {
+  const navie = navieProvider(NOP, NOP, NOP);
+  return new Promise((resolve, reject) => {
+    navie
+      .on('token', (response: string) =>
+        resolve(JSON.parse(response) as NavieRpc.V1.Suggest.Response)
+      )
+      .on('error', reject)
+      .ask(threadId, '@suggest /nocontext /nohelp', undefined, undefined)
```

Comment:
I am pretty sure that only `@explain` mode can auto-trigger `@help` but I guess `/nohelp` can't hurt.